pytorch/nlp/openmpi-wikitext2-rnn-lm/Readme.rst

```diff
@@ -0,0 +1,13 @@
+PyTorch Cifar-10 ResNet-20 Open-MPI Scaling
```

Comment:
This README should also be updated to the according model (here LSTM language model)